app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -1,37 +1,35 @@
+<li>
+  <a class="c-link c-link--block c-link--icon-left" href="<%= admin_path %>" aria-current="<%= "page" if controller.controller_name == "overview" %>">
+    <%= crayons_icon_tag("stack-line", class: "c-link__icon") %>
+    Overview
+  </a>
+</li>
```

Comment:
@Ridhwana it feels naughty to me that this link is the special snowflake being outside admin_menu.rb but I'm not sure how to add it there.. Thoughts?